public/locales/en/signup.json

```diff
@@ -76,7 +76,7 @@
   "labelParticipantAmount": "Number of registrants",
   "labelRegistration": "Registration",
   "labelUserConsent": "I agree to share information with the event organizer. The City of Helsinki does not use the information you provide for marketing purposes. <a href={{url}} target=\"_blank\" rel=\"noopener noreferrer\" aria-label=\"Data protection notice {{openInNewTab}}\">Data protection notice</a>",
-  "linkDataProtectionNotice": "https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Kayttajatunnusten%20hallinta.pdf",
+  "linkDataProtectionNotice": "https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Tietosuojaseloste%20Linked%20Registrations.pdf",
```

Comment:
This link gives me a 404. I found the following through Google and that one appears to work:

`https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Linked%20Registration%20tietosuojaseloste.pdf`

Reply:
Great catch! The file location seems to be changed for some reason. Changed to point to the suggested file.